.github/workflows/build-prod.yml

```diff
@@ -49,32 +45,15 @@ jobs:
         uses: google-github-actions/setup-gcloud@v0
       - run: |-
           gcloud auth configure-docker
-
-      - name: 'Build docker image with database password and Version tag'
-        if: ${{ inputs.docker-database-build-arg }}
-        run: |-
-          docker build --build-arg PASSWORD=${{ secrets.DATABASE_PASSWORD }} --build-arg ACTIVE_PROFILE=prod --tag "$DOCKER_IMAGE_NAME:$VERSION" .
-          docker build --build-arg PASSWORD=root --build-arg ACTIVE_PROFILE=local --tag "$DOCKER_IMAGE_NAME:local_latest" .
           
       - name: 'Build docker image with Version tag'
-        if: ${{ !inputs.docker-database-build-arg }}
         run: |-
-          docker build --build-arg ACTIVE_PROFILE=prod --tag "$DOCKER_IMAGE_NAME:$VERSION" .
-          docker build --build-arg ACTIVE_PROFILE=local --tag "$DOCKER_IMAGE_NAME:local_latest" .
+          docker build --tag "$DOCKER_IMAGE_NAME:$VERSION" .
```

Comment:
We could benefit from tagging it with **latest** as well, so it can be used in local env without need to specify exact tag. Then it can be pushed with `--all-tags` without specifying $VERSION